AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -129,27 +129,14 @@ private async Task<object> BookNewHearing(BookingDetailsRequest newBookingReques
                 
             _logger.LogInformation("BookNewHearing - Attempting to send booking request to Booking API");
             
-            if (_featureToggles.UseV2Api())
-            {
-                var newBookingRequestV2 = newBookingRequest.MapToV2();
-                    
-                var hearingDetailsResponse = await _bookingsApiClient.BookNewHearingWithCodeAsync(newBookingRequestV2);
-
-                hearingId = hearingDetailsResponse.Id;
-                    
-                response = hearingDetailsResponse.Map();
-            }
-            else
-            {
-                var newBookingRequestV1 = newBookingRequest.MapToV1();
-
-                var hearingDetailsResponse = await _bookingsApiClient.BookNewHearingAsync(newBookingRequestV1);
-                    
-                hearingId = hearingDetailsResponse.Id;
-                    
-                response = hearingDetailsResponse.Map();
-            }
+            var newBookingRequestV2 = newBookingRequest.MapToV2();
```

Comment:
Please remove IFeatureToggles from the constructor and in the test setup

Reply:
done